.github/workflows/deploy-to-beta-by-push.yml

```diff
@@ -5,6 +5,7 @@ concurrency:
   cancel-in-progress: true
 
 on:
+  workflow_dispatch:
   push:
     branches:
       - main
```

Comment:
After merge to the main it will run probably. So, here you can remove push branches main trigger.